libraries/AP_AHRS/AP_AHRS_Logging.cpp

```diff
@@ -54,14 +54,14 @@ void AP_AHRS::Write_Attitude(const Vector3f &targets) const
     const struct log_Attitude pkt{
         LOG_PACKET_HEADER_INIT(LOG_ATTITUDE_MSG),
         time_us         : AP_HAL::micros64(),
-        control_roll    : (int16_t)targets.x,
-        roll            : (int16_t)roll_sensor,
-        control_pitch   : (int16_t)targets.y,
-        pitch           : (int16_t)pitch_sensor,
-        control_yaw     : (uint16_t)wrap_360_cd(targets.z),
-        yaw             : (uint16_t)wrap_360_cd(yaw_sensor),
-        error_rp        : (uint16_t)(get_error_rp() * 100),
-        error_yaw       : (uint16_t)(get_error_yaw() * 100),
+        control_roll    : targets.x,
+        roll            : degrees(roll),
+        control_pitch   : targets.y,
+        pitch           : degrees(pitch),
+        control_yaw     : wrap_360_cd(targets.z),
+        yaw             : wrap_360(degrees(yaw)),
+        error_rp        : get_error_rp(),
```

Comment:
These last three lines are really not appropriate for this message as this is supposed to be the logging of the attitude controller.   I would like to remove the last three lines and move this logging to the AC_AttitudeControl library. This would let us optimise this log entry for what it is supposed to be for, monitoring the Attitude Control.

```
 error_rp        : get_error_rp(),
 error_yaw       : get_error_yaw(),
active          : uint8_t(active_EKF_type()),
```

Reply:
https://github.com/ArduPilot/ardupilot/pull/28026